migrations/20230309234219-add-yubikeyDeviceId-to-user-table.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn('Users', 'yubikeyDeviceId', {
+      type: Sequelize.STRING,
+      allowNull: true,
+      defaultValue: null,
+      unique: true,
```

Comment:
I don't think we should make that one unique, it's a legit use case to share the same key for multiple accounts (e.g. I use the same one for my personal and OC Cloudflare account)

```suggestion
```